### PR TITLE
Update vendored deps

### PR DESCRIPTION
**Pull request by LK4D4**
Main point of this commit is to bring new godbus/dbus, which needed for
go-systemd v4 with TLS support for socket activation.

### CODE REVIEW

**Comment by LK4D4** [comment]
I don't think that failure is related.

File: Godeps/_workspace/src/github.com/codegangsta/cli/app_test.go [hunk 1]
```diff
@@ -1,867 +0,0 @@
-package cli
```
Comment (aboch) [inline Godeps/_workspace/src/github.com/codegangsta/cli/app_test.go]: Why is this file being removed ?

File: Godeps/_workspace/src/github.com/docker/docker/vendor/src/github.com/Azure/go-ansiterm/LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```
Comment (aboch) [inline Godeps/_workspace/src/github.com/docker/docker/vendor/src/github.com/Azure/go-ansiterm/LICENSE]: Was it intentional to pull dependencies from `docker/docker/vendor` ?

File: Godeps/Godeps.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
 	"ImportPath": "github.com/docker/libnetwork",
-	"GoVersion": "go1.4.2",
+	"GoVersion": "go1.5.1",
```
Comment (aboch) [inline Godeps/Godeps.json]: Is this change necessary for the updated packages ?

Comment (LK4D4) [inline Godeps/Godeps.json]: No, but it's version for Docker now, I think it makes sense to test libnetwork with same, because docker is sole user of libnetwork now.
